WcaOnRails/spec/models/round_spec.rb

```diff
@@ -19,4 +19,152 @@
       expect(round).to be_invalid_with_errors(format: ["'m' is not allowed for '333'"])
     end
   end
+
+  context "time limit" do
+    let(:competition) { FactoryGirl.create :competition, event_ids: %w(333 444bf 555bf) }
+    let(:round) { FactoryGirl.create :round, competition: competition, event_id: "333" }
+
+    let!(:four_blind_round) { FactoryGirl.create :round, competition: competition, event_id: "444bf", format_id: "3" }
+    let!(:five_blind_round) { FactoryGirl.create :round, competition: competition, event_id: "555bf", format_id: "3" }
+
+    it "defaults to 10 minutes" do
+      expect(round.time_limit).to eq(TimeLimit.new(centiseconds: 10*60*100, cumulative_round_ids: []))
+      expect(round.time_limit_to_s).to eq "10:00.00"
+    end
+
+    it "set to 5 minutes" do
+      round.update!(time_limit: TimeLimit.new(centiseconds: 5*60*100, cumulative_round_ids: ["333-1"]))
```

Comment:
Somehow in my mind I was imagining a round with a cumulative time limit for itself *and* with itself (ie: the same id twice in the array).
We will probably want validation or warning for this and what you mentioned, but I'm fine with doing it when the user will actually be able to input something!